velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -1030,6 +1032,132 @@ TpchPlan TpchQueryBuilder::getQ10Plan() const {
   return context;
 }
 
+TpchPlan TpchQueryBuilder::getQ11Plan() const {
+  std::vector<std::string> partsuppColumnsSubQuery = {
+      "ps_availqty", "ps_suppkey", "ps_supplycost"};
+  std::vector<std::string> partsuppColumns = {
+      "ps_partkey", "ps_suppkey", "ps_availqty", "ps_supplycost"};
+  std::vector<std::string> supplierColumns = {"s_suppkey", "s_nationkey"};
+  std::vector<std::string> nationColumns = {"n_nationkey", "n_name"};
+
+  auto partsuppSelectedRowTypeSubQuery =
```

Comment:
Nit : Please remove the use of Selected in the names here to shorten them.

Reply:
Done, thanks for the suggestion!